content/press.adoc

```diff
@@ -79,6 +79,18 @@ to be used over the `headshot.png` image.
 
 If you have any questions about artwork, please ask one of the <<Press Contacts>>.
 
+
+== About the Continuous Delivery Foundation
+
+image::/images/sponsors/cdf-logo-color-knockout.png[alt="CDF Logo", float=right, width="164px"]
+
+The link:https://cd.foundation[Continuous Delivery Foundation (CDF)] serves as the vendor-neutral home of many of the fastest-growing projects for countinous integration/continuous delivery (CI/CD), including Jenkins, Jenkins X, Tekton, and Spinnaker,
+as well as fosters collaboration between the industry’s top developers, end users and vendors to further CI/CD best practices and industry specifications.
+
+Its mission is to grow and sustain projects that are part of the broad and growing continuous delivery ecosystem.
+The Foundation believes in the power of Continuous Delivery to empower developers and teams to produce high quality software more rapidly and the open source solutions collectively addressing the whole software delivery lifecycle. It fosters and sustains the ecosystem of open-source, vendor neutral projects through collaborations and interoperability, advocates this idea and encourages collaborations among practitioners to share and improve their practices.
+For more information about the foundation, please visit link:https://cd.foundation[its website].
+
```

Comment:
I'd prefer that we use the text from the [CDF About page](https://cd.foundation/about/) and that we include a comment noting where we found the text.  That will remind future editors of the source of the text.
```suggestion
// https://cd.foundation/about/
The link:https://cd.foundation[Continuous Delivery Foundation (CDF)] serves as the vendor-neutral home of many of the fastest-growing projects for continuous integration/continuous delivery (CI/CD).
It fosters vendor-neutral collaboration between the industry’s top developers, end users and vendors to further CI/CD best practices and industry specifications.
Its mission is to grow and sustain projects that are part of the broad and growing continuous delivery ecosystem.
```